src/arguments.py

```diff
@@ -0,0 +1,14 @@
+import typing as t
+from argparse import ArgumentParser
+
+
+def create_arguments() -> ArgumentParser:
+    ap = ArgumentParser(description='Module for migrate postgres database')
+    ap.add_argument('-p', '--path', required=True, help='Path to migrations folder')
+    ap.add_argument('-d', '--database', required=True, help='URI for connect to database')
```

Comment:
Take env variables from environment, and there is will be opportunity to use env variables as program args

for example:
```
MIGRATOR_MIGRATIONS_PATH
MIGRATOR_DB_URI
```